osfoffline/polling_osf_manager/osf_query.py

```diff
@@ -278,8 +277,9 @@ def make_request(self, url, method=None, params=None, expects=None, get_json=Fal
                 raise aiohttp.errors.BadStatusLine(response.status)
         elif 400 <= response.status < 600:
             content = yield from response.read()
-            error_message = '[status code: {}]:: {} @url '.format(str(response.status), str(content), str(url))
+            error_message = '[status code: {}]:: {} @url {}'.format(str(response.status), str(content), str(url))
             logging.error(error_message)
+            self.close()
             raise aiohttp.errors.HttpBadRequest(error_message)
```

Comment:
Ah- here is the place where the authorization error gets masked into a generic "Bad Request" exception.
(per our discussion of https://github.com/himanshuo/OSF-Offline/pull/13/files#diff-1d5902bdc5bcd3fa27c12b42ae6ff926R51)

Longterm this should be improved as it may mask edge cases. For now we'll leave it to ensure we can ship an MVP.